tests/unit/helpers/date-format-test.js

```diff
@@ -0,0 +1,10 @@
+import { momentFormat } from '../../../helpers/moment-format';
```

Comment:
I think this import just needs to be `date-format` instead of `moment-format` and then I believe the tests will pass.